layouts/partials/docs/edit.html

```diff
@@ -1,6 +1,15 @@
 {{ with .File }}
-    <a class="text-gray-600 hover:text-blue-700 text-xs" href="{{ $.Site.Params.repositoryURL }}/edit/{{ $.Site.Params.repositoryBranch }}/content/{{ .Path }}" target="_blank">
-        <i class="fab fa-github mr-2"></i>
-        Edit This Page
-    </a>
+<ul>
+   <li>
+      <a class="text-gray-600 hover:text-blue-700 text-xs" href="{{ $.Site.Params.repositoryURL }}/edit/{{ $.Site.Params.repositoryBranch }}/content/{{ .Path }}" target="_blank">
+	 <i class="fab fa-github mr-2"></i>
+         Edit This Page
+      </a>
+   <li>
+      <a class="text-gray-600 hover:text-blue-700 text-xs" href="{{ $.Site.Params.repositoryURL }}/issues/new?body=File: [{{ .Path }}]({{ $.Page.Permalink }})">
+        <i class="fab fa-check-square mr-2"></i>
```

Comment:
> How about maybe `fas fa-code-branch` for that one? @justinvp thoughts?

Actually, rather than `fas fa-code-branch`, how about `fas fa-edit` for the Edit icon?

Reply:
Works for me!